src/GovUk.Education.ExploreEducationStatistics.Admin/Services/AdminEventRaiser.cs

```diff
@@ -10,82 +12,49 @@ namespace GovUk.Education.ExploreEducationStatistics.Admin.Services;
     /// </summary>
-    public async Task OnThemeUpdated(Theme theme)
-    {
-        if (!eventGridClientFactory.TryCreateClient(
-                ThemeChangedEvent.EventTopicOptionsKey,
-                out var client))
-        {
-            return;
-        }
-
-        var evnt = new ThemeChangedEvent(theme);
-        
-        await client.SendEventAsync(evnt.ToEventGridEvent());
-    }
+    public async Task OnThemeUpdated(Theme theme) => 
+        await eventRaiser.RaiseEvent(new ThemeChangedEvent(theme));
 
     /// <summary>
     /// On Release Slug changed
     /// </summary>
-    public async Task OnReleaseSlugChanged(Guid releaseId, string newReleaseSlug, Guid publicationId, string publicationSlug)
-    {
-        if (!eventGridClientFactory.TryCreateClient(
-                ReleaseSlugChangedEvent.EventTopicOptionsKey,
-                out var client))
-        {
-            return;
-        }
-
-        var evnt = new ReleaseSlugChangedEvent(releaseId, newReleaseSlug, publicationId, publicationSlug);
-        await client.SendEventAsync(evnt.ToEventGridEvent());
-    }
+    public async Task OnReleaseSlugChanged(
+        Guid releaseId,
+        string newReleaseSlug,
+        Guid publicationId,
+        string publicationSlug) =>
+        await eventRaiser.RaiseEvent(new ReleaseSlugChangedEvent(releaseId, newReleaseSlug, publicationId, publicationSlug));
 
     /// <summary>
     /// On Publication Changed
     /// </summary>
-    public async Task OnPublicationChanged(Publication publication)
-    {
-        if (!eventGridClientFactory.TryCreateClient(
-                PublicationChangedEvent.EventTopicOptionsKey,
-                out var client))
-        {
-            return;
-        }
+    public async Task OnPublicationChanged(Publication publication) =>
+        await eventRaiser.RaiseEvent(new PublicationChangedEvent(publication));
 
-        var evnt = new PublicationChangedEvent(publication);
-        
-        await client.SendEventAsync(evnt.ToEventGridEvent());
-    }
-    
     /// <summary>
     /// On Publication Latest Published Release Reordered.
     /// It is assumed that the publication LatestPublishedReleaseVersionId has a value assigned to it.
     /// If it is null, then no event will be raised.
     /// </summary>
-    public async Task OnPublicationLatestPublishedReleaseReordered(Publication publication, Guid previousLatestPublishedReleaseVersionId)
+    public async Task OnPublicationLatestPublishedReleaseReordered(
+        Publication publication,
+        Guid previousLatestPublishedReleaseVersionId)
     {
-        if (!eventGridClientFactory.TryCreateClient(
-                PublicationLatestPublishedReleaseReorderedEvent.EventTopicOptionsKey,
-                out var client))
-        {
-            return;
-        }
-
         // Should the publication not have a latest published release version for some reason, do nothing.
         if (publication.LatestPublishedReleaseVersionId is null)
```

Comment:
It would probably be a good idea to add `#nullable enable` to `AdminEventRaiser` until we enable the nullable context project-wide.